content/rationale.md

```diff
@@ -0,0 +1,85 @@
+title: Why complate?
+
+Why complate?
+=============
+
+## Abstraction
```

Comment:
FWIW, I prefer underlining for both H1 and H2. I also add two leading blank lines to all headings for salience. Not sure whether my opinion actually matters here though; I probably won't be the one maintaining this.

Reply:
I just learned today that underline works for h1 and h2. What do you mean by two leading blank lines? Before the heading?

Reply:
Yeah:

```
Why complate?
=============


Abstraction
-----------

When we are developing our frontend components, we can use complate markup
abstractions ("macros") as an abstraction layer for our component. …


Composition
-----------

complate allows us to write components and then explicitly compose them
together. …
```

That makes them stand out more when reading/editing the source.